docs/sections/pipeline_samples/tutorials/generate_preference_dataset.ipynb

```diff
@@ -0,0 +1,427 @@
+{
```

Comment:
<p>can we add this to distilabel-internal-testing?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/distilabel/pull/917/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>